srcpkgs/deadbeef/template

```diff
@@ -1,37 +1,39 @@
 # Template file for 'deadbeef'
 pkgname=deadbeef
-version=1.8.8
-revision=3
+version=1.9.6
+revision=1
 create_wrksrc=yes
```

Comment:
```suggestion
```

Reply:
think this is still needed

```
=> deadbeef-1.9.6_1: running post_extract ...
mv: cannot stat 'deadbeef-1.9.6/*': No such file or directory
=> ERROR: deadbeef-1.9.6_1: post_extract: 'mv deadbeef-${version}/* .' exited with 1
=> ERROR:   in post_extract() at srcpkgs/deadbeef/template:46
```

Reply:
not if you apply the other comments (skip extraction and vsrcextract)